template.json

```diff
@@ -77,6 +77,10 @@
           "rec": {
             "value": 0,
             "label": "WITCHER.Actor.CoreStat.Rec"
+          },
+          "wt": {
```

Comment:
et can be confusing, should be  "woundTreshold"

Reply:
Fixed